tests/test_jsonchecker.py

```diff
@@ -38,6 +38,11 @@ async def test_check(self):
                         sha256="0000000000000000000000000000000000000000000000000000000000000000"
                     ),
                 )
+            elif data.filename == "jq-1.4.tarball_url.tar.gz":
```

Comment:
Should match `dest-filename` in the manifest, I suppose:
```suggestion
            elif data.filename == "jq-1.4.tarball.tar.gz":
```